src/models/Organization.ts

```diff
@@ -63,14 +73,14 @@ export function createAssociations(): void {
   Organization.belongsToMany(User, {
     as: "users",
     foreignKey: { name: "ipaCode", field: "organizationIpaCode" },
-    otherKey: { name: "fiscalCode", field: "userFiscalCode" },
+    otherKey: { name: "email", field: "userEmail" },
```

Comment:
Why the name is _otherKey_ and not _primaryKey_ ?

Reply:
For sequelize, `otherKey` represents the Primary Key of the target model of the Belongs-To-Many association. From [sequelize documentation](https://sequelize.org/master/manual/associations.html#belongs-to-many-associations):

> `foreignKey` will allow you to set **source model** key in the **through** relation. `otherKey` will allow you to set **target model** key in the **through** relation.
>
> ```javascript
> User.belongsToMany(Project, { as: 'Tasks', through: 'worker_tasks', foreignKey: 'userId', otherKey: 'projectId'})
> ```

Reply:
ok